### ISSUE TITLE
Document workaround for getting table-size in Citus

### ISSUE THREAD

**Issue by sumedhpathak** [citusdata/citus_docs#231]
From @marcocitus noted in issue: https://github.com/citusdata/citus/issues/631.

Workaround:

```sql
CREATE OR REPLACE FUNCTION citus_table_size(table_name regclass)
RETURNS bigint LANGUAGE plpgsql
AS $function$
DECLARE
        table_size bigint;
BEGIN
        PERFORM master_update_shard_statistics(shardid)
        FROM pg_dist_shard
        WHERE logicalrelid = table_name;

        SELECT sum(shard_size) INTO table_size
        FROM (
                SELECT max(shardlength) AS shard_size
                FROM pg_dist_shard JOIN pg_dist_shard_placement USING (shardid)
                WHERE logicalrelid = table_name AND shardstate = 1
                GROUP BY shardid
        ) shard_sizes;

        RETURN table_size;
END;
$function$;

SELECT logicalrelid AS Name, pg_size_pretty(citus_table_size(logicalrelid)) AS Size
FROM pg_dist_partition;

            name            |    size    
----------------------------+------------
 articles_hash              | 16 kB
 articles_single_shard_hash | 0 bytes
 test                       | 8192 bytes
(3 rows)
```

Edit: update to newer version

**Comment by metdos** [comment]
We should use max() instead of min() or eliminate invalid shards in the query.

This issue was brought up in citus-user.
https://groups.google.com/forum/#!msg/citus-users/n4N9gWz_Ssk/lUVdCP1TBwAJ

**Comment by marcocitus** [comment]
Was already fixed in #631, updated Sumedh's comment.

**Comment by velioglu** [comment]
3 UDFs are implemented to get the size of distributed tables. 

- citus_total_relation_size(relation_name) : Returns a distributed table and its indexes' total relation size. (b) Adds up total sizes on primary nodes. (c) If the function can't connect to a node, it errors out. (d) Works with shard replication factor equals to 1 and reference tables.

- citus_relation_size(relation_name) : Return size of the table excluding the size of the Free Space Map(FSM) and Visibility Map(VM).

- citus_table_size(relation_name) : Returns a distributed table's total relation size including size of FSM and VM, but excluding size of indexes.

Explanation of corresponding UDFs on postgresql : https://www.postgresql.org/docs/9.6/static/functions-admin.html#FUNCTIONS-ADMIN-DBSIZE

**Example SQL **

Examples of these UDFs can be found on : https://github.com/citusdata/citus/blob/master/src/test/regress/expected/multi_size_queries.out 

PR for the implementation : https://github.com/citusdata/citus/pull/1267
